### ISSUE TITLE
Enable keyboard navigation for Scratch GUI

### ISSUE THREAD

**Issue by quachtina96** [LLK/scratch-gui#902]
### Expected Behavior
You should be able to tab to different parts of the UI including but not limited to:

- between Blocks, Costumes, and Sounds
- Editor to sprite/backdrop
- Editor to asset panel
- Editor to stage/output

I'm not sure exactly what a logical flow for navigation would look like, but one approach involves prioritizing core UI like the green flag and the stop buttons and the block editor. A new/basic user will need to at the very least use the blocks and the green flag and the red stop button.  

### Actual Behavior
Currently, tabbing only allows you to navigate to specific elements--many buttons are ignored. There is no structure relating to prioritization of functions that are most common. The navigation of elements included in the tab order goes from top to bottom, right to left.

### Discussion Please!
Would love to hear ideas about what functions or UI to prioritize are what a keyboard navigation flow through the GUI could look like!

**Comment by towerofnix** [comment]
>Would love to hear ideas about what functions or UI to prioritize are what a keyboard navigation flow through the GUI could look like!

[Snap<i>!</i>](https://snap.berkeley.edu/) has a pretty cool keyboard editing feature for writing scripts with your keyboard.. probably not something to prioritize, but perhaps fun to think about! Scroll to page 96 (section: "Keyboard Editing") in the [Reference Manual](http://snap.berkeley.edu/SnapManual.pdf) for documentation.

Here's a series of screenshots (from the manual) which shows navigating a block in action (explanation in the manual):

![Navigating a block with keyboard editing](https://user-images.githubusercontent.com/9948030/33001418-ba8dc1e4-cda5-11e7-921c-445fed0b6398.png)

---

It'd be worth making the toolbar (scratch logo / "save" and "load" buttons, at the moment) navigatable via the keyboard, though not entirely necessary if there's obvious keybindings for those (e.g. control-S, control-O -- though I've seen some browsers are resistant to having those particular keys being overridden).

Add Sprite and Add Backdrop are both definitely important; without these, there's no way to create a new sprite or backdrop.

Within the block editor, navigating the category list would be handy (up/down arrows probably). Though maybe that should be tracked in scratch-blocks.
